.ci/checkTranslator.sh

```diff
@@ -123,8 +123,12 @@ usage () { (( $# > 0 )) && err "$*"; err "Usage: $0 <translator.js>"; }
 main() {
 
     # Add './node_modules/.bin to PATH for jsonlint
-    PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/node_modules/.bin:"$PATH"
+    PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/node_modules/.bin:"$PATH"
```

Comment:
It looks for me that you maybe can delete another pair of `"` after the `cd`. No? At least it then still seems to work fine for me.